core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/ClusterDbCertRotationHandler.java

```diff
@@ -0,0 +1,37 @@
+package com.sequenceiq.cloudbreak.reactor.handler.cluster;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.eventbus.Event;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.ClusterDbCertRotationRequest;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.ClusterDbCertRotationResult;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.ExceptionCatcherEventHandler;
+import com.sequenceiq.flow.reactor.api.handler.HandlerEvent;
+
+@Component
+public class ClusterDbCertRotationHandler extends ExceptionCatcherEventHandler<ClusterDbCertRotationRequest> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClusterDbCertRotationHandler.class);
+
+    @Override
+    public String selector() {
+        return EventSelectorUtil.selector(ClusterDbCertRotationRequest.class);
+    }
+
+    @Override
+    protected Selectable defaultFailureEvent(Long resourceId, Exception e, Event<ClusterDbCertRotationRequest> event) {
+        return new ClusterDbCertRotationResult("", e, event.getData());
```

Comment:
No, I think we need some different failure event here (e.g. `StackFailureEvent`). I think this particular approach works only because `ClusterDbCertRotationResult` is now an instance of `ClusterPlatformResult` (incorrectly). If you have a look, its selector is defined as follows:
```
    @Override
    public String selector() {
        return status == EventStatus.OK ? EventSelectorUtil.selector(getClass()) : EventSelectorUtil.failureSelector(getClass());
    }
```
So it is basically status-aware and returns either a success or a failure selector, just like `CloudPlatformResult` in `cloud-reactor-api`.

Reply:
Let us leave it as is for now; can be investigated & refactored later if needed.